src/index.tsx

```diff
@@ -7,7 +7,6 @@ import Bugsnag from "@bugsnag/js";
 import BugsnagPluginReact from "@bugsnag/plugin-react";
 import LogRocket from "logrocket";
 // eslint-disable-next-line no-restricted-imports
-import mixpanel from "mixpanel-browser";
```

Comment:
If you're moving the line it relates to, remove the associated comment as well.

Reply:
fix in 53838073e2551675ad8db79c5d249367d64c7700